dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/h12/http2/GenericHttp2ServerTransportListener.java

```diff
@@ -164,7 +165,7 @@ protected void onMetadataCompletion(Http2Header metadata) {
     @Override
     protected void onDataCompletion(Http2InputMessage message) {
         if (message.isEndStream()) {
-            serverCallListener.onComplete();
```

Comment:
Why remove it?

Reply:
Closing the decoder triggers the onComplete method of the serverCallListener.
![image](https://github.com/apache/dubbo/assets/18413695/3a040fcc-70a6-4deb-b1aa-8ea563a7a04c)